### DISCUSSION TITLE
Hide tab numbers (⌘number) in macOS

### DISCUSSION THREAD

**Discussion by giuliomagnifico** [ghostty-org/ghostty#3948] [Q&A]
Hi, is there a setting to hide the tab numbers (⌘1, ⌘2, ⌘3, etc) on the tabs? I haven't found it anywhere. Thanks

**Comment by edrozenberg** [comment]
Would be nice to have the option. Don't see anything like that atm in https://ghostty.org/docs/config/reference.

Other terminals (Mac Terminal.app, Alacritty) do this by default - there is no CMD-# info on the tabs, but the shortcuts do work.

**Comment by mostpinkest** [comment]
This is not currently implemented. For reference, here is the section of code responsible for setting the tab's shortcut label: https://github.com/ghostty-org/ghostty/blob/72d085525b22d66468c5969a4d507a0fa68d4a04/macos/Sources/Features/Terminal/TerminalController.swift#L151-L180

**Comment by johan456789** [comment]
Yes. I would like that option too. Having that shortcut messes with my judgement of which tab I am in.
